java/maven/src/org/netbeans/modules/maven/problems/SanityBuildAction.java

```diff
@@ -97,6 +107,21 @@ public void run() {
                     String label = build_label(nbproject.getProjectDirectory().getNameExt());
                     config.setExecutionName(label);
                     config.setTaskDisplayName(label);
+                    
+                    MavenProxySupport mps = nbproject.getLookup().lookup(MavenProxySupport.class);
+                    if (mps != null) {
+                        ProxyResult res;
```

Comment:
this variable seems to be unused

Reply:
Hm, I'll better check the behaviour with 'cancel' in the dialog; thanks !

Reply:
fixed in 5d31b280c9d8